integration/source/integration/api.d

```diff
@@ -24,14 +24,14 @@ unittest {
     const database = Database(influxURL, "myspecialDB");
     scope(exit) database.drop;
 
-    database.insert(Measurement("cpu", ["tag1": "foo"], ["temperature": "42"]));
-    database.insert(Measurement("cpu", ["tag1": "foo"], ["temperature": "68"]));
+    database.insert(Measurement("cpu", ["tag1": "foo"], ["temperature": 42]));
+    database.insert(Measurement("cpu", ["tag1": "foo"], ["temperature": 68]));
 
     {
         const response = database.query("SELECT * from cpu");
         const result = response.results[0];
         const series = result.series[0];
-        series.rows.length.shouldEqual(2);
+        series.rows.length.shouldEqual(1);
```

Comment:
The old implementation was wrong. 1 is the correct value because tags are the same and no timestamps.